packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -3432,7 +3432,7 @@ const tests = {
           React.useEffect(() => {
             console.log(props.foo);
           }, []);
-          React.useCustomEffect(() => {
+          customModule.useCustomEffect(() => {
```

Comment:
Could we duplicate this test instead so that both React and other modules are covered separately?

Reply:
I could definitely add the following test case back in:

```js
                  React.useCustomEffect(() => {
                    console.log(props.foo);
                  }, []);
```

But as mentioned in the PR description, the test case actually fails now, because it'd now expect the linter to error due to props.foo not having been included in the dependency list, whereas previously it was expecting no errors (which feels wrong, but I could be missing some nuance here, would definitely appreciate input from folks with more context). 

I could add the test case back in with an error expectation if people feel the test case itself still adds value.